src/docs/release/breaking-changes/scrollable_alert_dialog.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Scrollable AlertDialog
+description: AlertDialog should scroll automatically when it overflows.
+---
+
+# Scrollable AlertDialog
+
+## Context
+
+Currently, when an AlertDialog widget’s contents are too tall, they can end up
+overflowing and can be clipped. This causes the following issues:
+
+1. The content overflows and there is no way to view the rest of the content.
+2. Most alert dialogs have buttons beneath its contents to prompt users for
+actions. However, if the contents overflow and the buttons are obscured, users
+may be unaware of the existence of these buttons.
+
+## Description of change
+
+The current code simply has the title widget and content widgets appear consecutively in a Row widget:
+
+```dart
+Column(
+  mainAxisSize: MainAxisSize.min,
+  crossAxisAlignment: CrossAxisAlignment.stretch,
+  children: <Widget>[
+    if (title != null)
+      Padding(
+        padding: titlePadding ?? EdgeInsets.fromLTRB(24.0, 24.0, 24.0, content == null ? 20.0 : 0.0),
+        child: DefaultTextStyle(
+          style: titleTextStyle ?? dialogTheme.titleTextStyle ?? theme.textTheme.title,
+          child: Semantics(
+          child: title,
+          namesRoute: true,
+          container: true,
+          ),
+        ),
+      ),
+    if (content != null)
+      Flexible(
+        child: Padding(
+        padding: contentPadding,
+        child: DefaultTextStyle(
+          style: contentTextStyle ?? dialogTheme.contentTextStyle ?? theme.textTheme.subhead,
+          child: content,
+        ),
+      ),
+    ),
+    // ...
+  ],
+);
+```
+
+The proposed change would basically wrap both widgets in a
+SingleChildScrollView above the button bar, making both
+widgets part of the same scrollable while exposing the
+button bar at the bottom of the dialog:
+
+```dart
+Column(
+  mainAxisSize: MainAxisSize.min,
+  crossAxisAlignment: CrossAxisAlignment.stretch,
+  children: <Widget>[
+    if (title != null || content != null)
+      SingleChildScrollView(
+        child: Column(
+        mainAxisSize: MainAxisSize.min,
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+         children: <Widget>[
+           if (title != null)
+             titleWidget,
+             if (content != null)
+             contentWidget,
+         ],
+       ),
+     ),
+   // ...
+  ],
+),
+```
+
+## Migration guide
```

Comment:
please make sure to include before/after code snippets

Reply:
This one is tough to do because the changes can vary depending on what is happening for the app. For example, the semantic golden tests would be pretty straightforward since you'd just have to update the goldens. 

However, for layout breakages, I'm unsure what kind of layout issues might arise since I could not cause any myself, but I'm sure I saw an internal test break due to layout changes, although I'm unclear what caused it.